pkg/build/ignore/docker/dockign.go

```diff
@@ -0,0 +1,116 @@
+package docker
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/openshift/source-to-image/pkg/api"
+)
+
+type DOCKIGN struct {
+	Config *api.Config
+}
+
+func New(req *api.Config) *DOCKIGN {
+	d := &DOCKIGN{
+		Config: req,
+	}
+	return d
+}
+
+func (b *DOCKIGN) Ignore(config *api.Config) {
+	path := filepath.Join(config.WorkingDir, "upload", "src", api.DefaultIgnoreFile)
```

Comment:
yeah I think where I was suggesting we go is something closer to:
filepath.Join(config.WorkingSourceDir,api.DefaultIgnoreFile)

where config.WorkingSourceDir gets built up in a single spot elsewhere.